README

```diff
@@ -5,4 +5,5 @@ Requires gradle and IntelliJ IDEA for building.
 Build instructions:
 - edit build.gradle and change values of 'ideaInstallationPath' and 'ideaJdk' to be in sync with your environment
 - invoke 'gradle build' from command line to build
-- invoke 'gradle idea' from command line to generate IDEA specific files - provides possibility to be opened as IDEA project.
\ No newline at end of file
+- invoke 'gradle idea' from command line to generate IDEA specific files - provides possibility to be opened as IDEA project.
+- in IDEA, select 'File' - 'Open' - ${YOUR_DIR}/Builder-Generator.ipr, donot use 'File' - 'Import' !!!!
```

Comment:
Yes, that's really necessary for people who are not familiar with plugin development, it takes me several hours to find the right way. Please ignore my modification.